pkg/addons/addons.go

```diff
@@ -141,15 +141,20 @@ func enableOrDisableAddon(cc *config.ClusterConfig, name string, val string) err
 		}
 	}
 
-	// to match both ingress and ingress-dns adons
-	if strings.HasPrefix(name, "ingress") && enable && driver.IsKIC(cc.Driver) && runtime.GOOS != "linux" {
-		exit.UsageT(`Due to {{.driver_name}} networking limitations on {{.os_name}}, {{.addon_name}} addon is not supported for this driver.
+	// to match both ingress and ingress-dns addons
+	if strings.HasPrefix(name, "ingress") && enable {
+		if driver.IsKIC(cc.Driver) && runtime.GOOS != "linux" {
+			exit.UsageT(`Due to networking limitations of driver {{.driver_name}} on {{.os_name}}, {{.addon_name}} addon is not supported.
```

Comment:
I changed this text because I thought the text of
"Due to none networking limitations on linux..."

Could be confusing to some minikube users. Let me know if this should be reverted/chaged.

Alternatively, the driver name could be quoted to prevent confusion:
```suggestion
			exit.UsageT(`Due to "{{.driver_name}}" networking limitations on {{.os_name}}
```

Reply:
This wording seems fine to me.